ext/afform/admin/ang/afGuiEditor/afGuiSearch.html

```diff
@@ -23,7 +23,7 @@
             </li>
           </ul>
         </div>
-        <input ng-if="filter.mode === 'routeParams'" class="form-control" ng-model="filter.value" />
+        <input ng-if="filter.mode === 'routeParams'" class="form-control" ng-model="filter.value" title="{{:: ts('Append %1 to the URL to pass a parameter value to the form' , {1: '\'#/?' + filter.value + '=value\''}) }}" />
```

Comment:
Probably safer to remove the `::` after the opening brackets since the `ts()` uses params that could theoretically change.
(FYI in AngularJS templates, `{{:: foo() }}` means only evaluate the expression once, ever, whereas `{{ foo() }}` means re-evaluate whenever something changes.

Reply:
Thanks @colemanw, have removed the double colons.